scripts/hudson/quickstart.bat

```diff
@@ -37,7 +37,12 @@ set WORKSPACE=%OLDWORKSPACE%
 rem git checkout 4.17
 rem call build.bat clean install -DskipTests
 
-for /f "usebackq delims=<,> tokens=3" %%i in (`findstr "version.org.wildfly.wildfly-parent" blacktie\pom.xml`) do @set WILDFLY_MASTER_VERSION=%%i
+set WILDFLY_MASTER_VERSION=
+for /f "usebackq delims=<,> tokens=3" %%i in (`findstr "<version>1" jboss-as\pom.xml`) do (
```

Comment:
I'm not sure how does this work, but just wanted to check if that "1" is there on purpose?